py-polars/polars/dataframe/frame.py

```diff
@@ -10398,6 +10398,27 @@ def update(
             .collect(_eager=True)
         )
 
+    def count(self) -> DataFrame:
+        """
+        Return the number of non-null elements in all the columns.
```

Comment:
Nitpicky, but I think this reads a bit nicer, otherwise I might expect an integer with the total null count across columns:

```suggestion
        Return the number of non-null elements for each column.
```

(same for the other docstrings & rust side docstring)

Reply:
Haha, that's what I was doubting about as well